services/api-server/src/simcore_service_api_server/services/rabbitmq.py

```diff
@@ -4,22 +4,28 @@
 from servicelib.rabbitmq import RabbitMQClient, wait_till_rabbitmq_responsive
 from settings_library.rabbit import RabbitSettings
 
+from ..services.log_streaming import LogDistributor
+
 _logger = logging.getLogger(__name__)
 
 
 def setup_rabbitmq(app: FastAPI) -> None:
     settings: RabbitSettings = app.state.settings.API_SERVER_RABBITMQ
     app.state.rabbitmq_client = None
-    app.state.rabbitmq_rpc_server = None
+    app.state.log_distributor = None
```

Comment:
wondering. it is a rpc_server/client or only for logs?

Reply:
This one is only for logs. The `LogDistributor` is a small wrapper around the `RabbitMQClient` which puts the incoming log messages into the correct queue so that they can be sent back to the right client.